test/SILOptimizer/peephole_thick_to_objc_metatype.sil

```diff
@@ -115,3 +117,47 @@ bb0(%0 : $T):
   strong_release %0 : $T
   return %3 : $@thick T.Type
 }
+
+// CHECK-LABEL: sil @$test_peephole_objc_to_thick_to_objc
```

Comment:
It is good to include the ':' after the name so that you know that the type part of the SIL function declaration begins. This prevents by mistake matching function name prefixes. I am not saying this will happen with this function specifically, but this is just a good rule to follow to define away the bug.